cypress/integration/expert.spec.js

```diff
@@ -0,0 +1,47 @@
+// Tests the functionality from the Community member perspective
+
+describe("Community members", () => {
+    beforeEach(() => {
+        cy.login("expert", "1234567890");
+        cy.get('#djHideToolBarButton').click();
+    })
+
+    it("can mark a translation as good or bad", () => {
+        cy.visit(Cypress.env('home'));
+
+        let customGreen = null;
+
+        cy.window()
+            .then((win) => {
+                customGreen = win.getComputedStyle(win.document.documentElement).getPropertyValue('--custom-pale-green');
```

Comment:
This is never used. I’m glad we didn’t have to go this route.

Reply:
Oh my goodness, me too! Taking this out now